mlflow/tensorflow/__init__.py

```diff
@@ -805,7 +856,37 @@ def log_model_without_starting_new_run():
                         tf_meta_graph_tags=[tag_constants.SERVING],
                         tf_signature_def_key="predict",
                     )
-                    save_model(path=local_path, mlflow_model=mlflow_model, **save_model_kwargs)
+
+                    input_example = None
+                    signature = None
+                    if log_models:
```

Comment:
Nice! If `log_models` is `False`, we shouldn't be logging the model at all. Looks like the previous version of the code didn't respect this flag.

Can we move the `save_model` and `MlflowClient.log_artifacts()` calls under this `if log_models` condition as well?

Reply:
done. thx